src/app/checkout/step-3/step-3.component.js

```diff
@@ -86,6 +93,22 @@ class Step3Controller{
         });
   }
 
+  getNextDrawDate(){
+    this.commonService.getNextDrawDate().subscribe(nextDrawDate => {
+      this.nextDrawDate = nextDrawDate;
+    });
+  }
+
+  updateGiftStartMonth(item, month){
+    item.config['recurring-start-month'] = month;
+
+    this.cartData = null;
+    this.cartService.editItem( item.uri, item.productUri, item.config ).subscribe( () => {
+      this.$scope.$emit( cartUpdatedEvent );
```

Comment:
Do we want the nav cart to open? Is that all this does? If you keep this I think you need to create a spy. `spyOn(self.controller.$scope, 'emit');`

Reply:
No, I'll remove it.